src/OrbitQt/HistogramWidget.cpp

```diff
@@ -28,6 +30,11 @@
 #include "OrbitBase/Logging.h"
 #include "Statistics/Histogram.h"
 
+const QColor kBackgroundColor(QString::fromStdString("#323232"));
+constexpr size_t kBarColorsCount = 2;
+const std::array<QColor, kBarColorsCount> kBarColors = {QColor(QString::fromStdString("#2A82DA")),
```

Comment:
I would define #2A82DA as a constant.
Ex: constexpr QColor kColorCyan = ...

Same with the other color.

Reply:
Do you mean, there should be `constexpr std::string = "#...."` ?
What's about #323232 ?

On the one hand, it will name the magic string constant, on the other hand, it introduces a variable that is used only once right after it's defined. Moreover, it's used to create another constant of an extremely similar name. I'm not sure about this.

Reply:
no, I mean using `constexp Qcolor` directly

We are creating a constant name only to let developers to understand what this color means without having to run Orbit directly. In general there is no magic constants in Orbit. https://abseil.io/tips/140

Reply:
Still not sure what you mean, sorry. Can you give an example? All constructors of `QColor` seem to be non-constexpr.

Reply:
Oh, sorry. I'm always thinking about const and constexpr as they were the same. I'm only worried about giving them a name in this context. It's doing exactly the same you did with kBackgroundColor, both:
const QColor kBlueColor(QString::fromStdString("#2A82DA"));
or
const QColor kBlueColor{42, 130, 218};

are ok for me. 

In the future I would like to have a place where we store and use all the colors, also to be a bit more standart with color selection.